yt_dlp/extractor/radiko.py

```diff
@@ -234,6 +236,7 @@ def _real_extract(self, url):
         title = prog.find('title').text
         description = clean_html(prog.find('info').text)
         station_name = station_program.find('.//name').text
+        performer = prog.find('pfm').text
```

Comment:
wrap in `try_call` and inline in return

Reply:
Thank you for your feedback. I fixed it.